src/api/messagesFlags.js

```diff
@@ -1,14 +1,16 @@
-import { apiPost, Auth } from './apiFetch';
+/* @flow */
+import { Auth } from '../types';
+import { apiPost } from './apiFetch';
 
 export default (
   auth: Auth,
   messages: number[],
   op: string,
   flag: string,
-): number[] =>
+): mixed =>
```

Comment:
It will be `any` not mixed. Because our resFunc can return `any` here we alse have to have the same type.